src/tuplet.ts

```diff
@@ -310,17 +339,12 @@ export class Tuplet extends Element {
 
       // only draw the bracket if it has positive length
       if (line_width > 0) {
-        this.context.fillRect(this.x_pos, this.y_pos, line_width, 1);
-        this.context.fillRect(this.x_pos + this.width / 2 + width / 2 + 5, this.y_pos, line_width, 1);
-        this.context.fillRect(
-          this.x_pos,
-          this.y_pos + (this.location === Tuplet.LOCATION_BOTTOM),
-          1,
-          this.location * 10
-        );
-        this.context.fillRect(
+        ctx.fillRect(this.x_pos, this.y_pos, line_width, 1);
+        ctx.fillRect(this.x_pos + this.width / 2 + width / 2 + 5, this.y_pos, line_width, 1);
+        ctx.fillRect(this.x_pos, this.y_pos + +(this.location === Tuplet.LOCATION_BOTTOM), 1, this.location * 10);
+        ctx.fillRect(
           this.x_pos + this.width,
-          this.y_pos + (this.location === Tuplet.LOCATION_BOTTOM),
+          this.y_pos + +(this.location === Tuplet.LOCATION_BOTTOM),
```

Comment:
I'm not sure happening here?

Reply:
What was the intention in origin? I used the solution from @wassertim I also quite not get it summing a number and a boolean.

Reply:
I have changed it to a ?: construction. Easie to understand.

Reply:
Yeah, that's better -- not sure how the original version got that, definitely not readable.